src/createPackage/models/createPackageManager.ts

```diff
@@ -566,90 +368,6 @@ export class CreatePackageManager {
     }
   }
 
-  private generateTileGroups(polygon: Polygon | MultiPolygon, footprint: Polygon | MultiPolygon, zoom: number): ITileRange[] {
```

Comment:
please revert its deletion and put in comment, add:
// TODO: remove and replace with `generateTileGroups` that is commented, when multiple tasks for GPKG target is possible